src/figuresClasses.ts

```diff
@@ -1,11 +1,79 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
```

Comment:
You need to limit possible shapes (say using the union type)
```suggestion
  shape:'triagle' | 'circle' | 'rectangle';
```